types/duplexify/duplexify-tests.ts

```diff
@@ -0,0 +1,16 @@
+import * as duplexify from "duplexify";
```

Comment:
Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
@RyanCavanaugh Changed test to use `require`.